README.md

```diff
@@ -64,6 +64,7 @@ The instrumentations in this repo are:
 
 | Instrumentations Version | OpenTelemetry Core | OpenTelemetry Experimental | 
 | --- | --- | --- |
+| 0.34.x | ^1.3.0 | ^0.34.0 |
```

Comment:
I think it should be `^1.8.0` instead of `^1.3.0` this is the version of SDK core packages.